### DISCUSSION TITLE
Error "No such entity with cartID" in swissup_firecheckout module.

### DISCUSSION THREAD

**Discussion by akashpatelcp** [breezefront/community#321] [1. Help]
When we install the Firecheckout module, this error occurs. If we add a $0 price product, then  'No Payment information required' appears in the payment method section. After that, when we try to place the order, the 'No such entity with cartID' error is shown in the coupon code section.Order is successfully placed.I have attached the screenshot.

![Screenshot from 2024-08-27 13-38-53](https://github.com/user-attachments/assets/a5cc76f1-1655-436f-b831-71f9e162e84d)
![Screenshot from 2024-08-27 13-38-29](https://github.com/user-attachments/assets/3f6c3bdf-5bd8-4b18-b88c-a9c4696e6af1)
![Screenshot from 2024-08-27 13-34-44](https://github.com/user-attachments/assets/34b228b0-163c-40b0-ac6a-fdd0c4221951)

The fire checkout is installed in fresh magento 2.4.6-p6 version.There are no any customise

**Comment by roma84** [comment]
These discussions are for Breeze-related issues.
Please create a ticket on a site where you bought the Firecheckout module.
